ingestion/src/metadata/ingestion/source/dashboard/powerbi.py

```diff
@@ -188,4 +188,3 @@ def yield_dashboard_chart(
             except Exception as err:  # pylint: disable=broad-except
                 logger.debug(traceback.format_exc())
                 logger.error(err)
-                self.status.failure(chart["title"], repr(err))
```

Comment:
instead of not reporting this should we maybe run a `chart.get("title")` and default to the chart ID or something? The `repr` we can leave out as we already have the log

Reply:
I think instead of chart title we can directly use chart id here if a chart fails....Because for one of the users, in one chart chart['title'] itself was throwing an error and when it wen't to exception block it was again throwing the same exception. So the ingestion was breaking

Reply:
works for me! Thanks!

Reply:
@pmbrull also the`repr` is required here since the`status.failure' requires 2 arguments ( error_name, error_reason)

Reply:
thanks LGTM